react/SearchWrapper.tsx

```diff
@@ -147,13 +163,15 @@ const SearchWrapper: FC<SearchWrapperProps> = props => {
     const { products } = searchQuery
 
     const event = getPageEventName(products, params)
-
     const pageInfoEvent: SearchPageInfoEvent & PixelEvent = {
       event: 'pageInfo',
       eventType: event,
       accountName: account,
       pageTitle: title,
       pageUrl: window.location.href,
+      orderBy,
+      map: map?.split(','),
```

Comment:
Return an array of objects so we can improve it in the future with the values as well

```suggestion
      appliedFilters: map?.split(',').map(name => ({ name }),
```

Reply:
Example in the filter we could have

```
appliedFilters: [{ name: 'gender', value: 'men' }, { name: 'size', value: '42' }]
```